virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -0,0 +1,266 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-4-15-release-notes"]
+= {VirtProductName} release notes
+include::_attributes/common-attributes.adoc[]
+:context: virt-4-15-release-notes
+
+toc::[]
+
+[id="virt-inclusive-language"]
+== Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
+
```

Comment:
I looked a Chris Wright's message, and the person he refers to as the head of Diversity and Inclusion no longer works at Red Hat. It seems that she left in 2020.

Reply:
This probably doesn't address the issue Audrey pointed out, but it looks like we're supposed to include this module instead of pasting the text:
```
include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
```
source: [OCP docs contrib guide](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#adding-a-subsection-on-making-open-source-more-inclusive)